### DISCUSSION TITLE
Terminal indicating "server returned an error response"

### DISCUSSION THREAD

**Discussion by Tarreef** [Cyfrin/foundry-full-course-cu#3062] [Q&A]
**while trying to create a deployment on VScode terminal using "forge create"....**

```
forge create SimpleStorage --rpc-url HTTP://127.0.0.1:7545 --interactive

```

**i got the typical response requiring the private key to move on...**

```
[⠊] Compiling...
No files changed, compilation skipped
Enter private key:
```

**after pasting in the required private key from ganache i got and error....**

```
Error: server returned an error response: error code -32000: VM Exception while processing transaction: invalid opcode, data:
{"hash":null,"programCounter":23,"result":"0x","reason":null,"message":"invalid opcode"}
```

**_Please has anyone experienced the same error??_**

**Comment by satyaorz** [comment]
As far as i remember, now its not recommended to use Ganache as it is not maintained anymore, use Anvil instead.

**Reply by EngrPips** [reply]
Great heads-up.
